src/Contract/Transaction.purs

```diff
@@ -315,6 +357,18 @@ balanceAndLock { transaction, usedUtxos, balancerConstraints } = do
   void $ withUsedTxOuts $ lockTransactionInputs balancedTx
   pure balancedTx
 
+balanceAndLockUtxos
```

Comment:
Also, add some comment to this one, too. Public-facing API should have documentation comments.